<launch>

  <!-- main node -->
  <node pkg="volta_main" type="core_node" name="core_node" output="screen"/>
  <node pkg="volta_main" type="bluetooth_node" name="bluetooth_node" output="screen"/>
  <node pkg="volta_main" type="auto_navi_node" name="auto_navi_node" output="screen"/>
  <node pkg="turn_signal" type="turn_signal" name="turn_signal" output="screen"/>

  <node pkg="rosserial_python" type="serial_node.py" name="serial_node2" output="screen">
	<param name="port" value="/dev/ttyACM0" />
	<param name="baud" value="9600" />
  </node>



</launch>





